docusaurus/docs-internals/editor-controls/group-controls/order.md

```diff
@@ -0,0 +1,170 @@
+---
+toc_max_heading_level: 4
+---
+
+# Order
+
+The `order` control is designed to reorder items within a container, either horizontally or vertically. It consists of two arrows that allow users to change the position of an item.
+
+Example of the control with horizontal arrows:
+
+![Order Horizontal](/img/ui-controls/order-horizontal.png)
+
+Example of the control with vertical arrows:
+
+![Order Vertical](/img/ui-controls/order-vertical.png)
+
+### Parameters
+
+| Name               | Type                                        |      Default      | Description                                                                                                                                                                                                                                                        |
+|:-------------------|:--------------------------------------------|:-----------------:|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `id`               | `string`                                    |         -         | The identifier of the key where the `"order"` will save your data                                                                                                                                                                                                  |
+| `type`             | `string`                                    |         -         | Type should be `"order"` to use this control                                                                                                                                                                                                                       |
+| `position?`        | `number`                                    |         -         | The position of the control in toolbar                                                                                                                                                                                                                             |
+| `devices?`         | `"all"` \| `"desktop"` \| `"responsive"`    |      `"all"`      | Define the devices where the control will be rendered. `"all"` renders the control on all devices. `"desktop"` renders the control only on desktop devices. `"responsive"` renders the control on both tablet and mobile devices                                   |
+| `disabled?`        | `boolean`                                   |      `false`      | Configure the condition under which the control is disabled or enabled                                                                                                                                                                                             |
```

Comment:
didn't include the `label` 
![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/85360293/0a84a49c-3e71-4a7c-9a21-51fad07b213b)